integration-tests/tx-rollup-l2-address.spec.ts

```diff
@@ -82,13 +82,13 @@ CONFIGS().forEach(({lib, setup, protocol}) => {
       const pkh = await contract.storage()
       expect(pkh).toEqual("tz4QyWfEiv56CVDATV3DT3CDVhPaMKif2Ce8");
 
-      const methodCall = await contract.methods.default(pkh).send();
-      await methodCall.confirmation();
-      expect(methodCall.includedInBlock).toBeLessThan(Number.POSITIVE_INFINITY)
+      const ticketDeposit = await contract.methods.default(pkh).send();
+      await ticketDeposit.confirmation();
+      expect(ticketDeposit.includedInBlock).toBeLessThan(Number.POSITIVE_INFINITY)
       done();
     })
   })
-  describe(`mondaynet contract originations and method calls to test the type of tx_rollup_l2_address`, () => {
+  describe(`it contract originations and method calls to test the type of tx_rollup_l2_address`, () => {
```

Comment:
Is there a difference between these tests and the precedent, it seems to be duplication.